test/README.md

```diff
@@ -0,0 +1,12 @@
+# Test directory
+
+This is default directory for salgenerator usage.
+If you wish to use another location, it can be set using the
+environment variable SAL\_WORK\_DIR.
+
+If you wish to generate SAL language bindings and libraries in some other directory, you should:
+
+```bash
+export SAL_WORK_DIR=$(pwd)
+cp $LSST_SDK_INSTALL/test/VERSION .
```

Comment:
I don't have a VERSION file in my test directory, nor anything that looks remotely relevant.
```
$ ls test
python  README
```

Reply:
test/README.md was deleted.